apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -432,6 +432,7 @@ func (s *bundleSuite) TestGetChangesMapArgsBundleEndpointBindingsSuccess(c *gc.C
 }
 
 func (s *bundleSuite) TestExportBundleFailNoApplication(c *gc.C) {
+	c.Skip("We are dropping support of bundle in 4.0.")
```

Comment:
todo: remove these tests, they can be added back when the code is.

There is no official statement that we are dropping bundle support in 4.0. We are deprioritizing until it can be re-written to not use model migration. The effort is low priority at this time.

Reply:
Agreed, I think its incorrect to say that we are dropping bundle support in 4.0 at the moment.

Reply:
We can probably just drop exporting bundles @hmlanigan. But let's chat about that in the sync next week.